wdl/MainVcfQc.wdl

```diff
@@ -858,3 +886,111 @@ task SanitizeOutputs {
   }
 }
 
+
+# Identify all duplicates in a single file
+task IdentifyDuplicates {
+  input {
+    String prefix
+    File vcf
+    String sv_pipeline_qc_docker
+    File? custom_script
+    RuntimeAttr? runtime_attr_override
+  }
+
+  # File default_script = "/src/sv-pipeline/scripts/merge_duplicates.py"
+  # File active_script = select_first([custom_script, default_script])
+
+  String vcf_basename = basename(vcf, ".vcf.gz")
+  String full_prefix = "~{prefix}.~{vcf_basename}"
+
+  RuntimeAttr runtime_default = object {
+    mem_gb: 3.75,
+    disk_gb: 2 + ceil(size(vcf, "GiB")),
```

Comment:
```suggestion
    disk_gb: 10 + ceil(size(vcf, "GiB")),
```
2GB is pretty small. VM disk speed scales with its size, so we usually set to at least 10.

Reply:
Updated accordingly - thanks for the tip.